pkg/graph/resolvers/model_plan.go

```diff
@@ -108,9 +120,63 @@ func ModelPlanCreate(ctx context.Context, logger *zap.Logger, modelName string,
 		return nil, err
 	}
 
+	if emailService != nil && emailTemplateService != nil {
+		err = sendModelPlanCreatedEmail(
+			ctx,
+			emailService,
+			emailTemplateService,
+			emailService.GetConfig().GetDevTeamEmail(),
+			createdPlan,
+		)
+		if err != nil {
+			logger.Error("failed to send model plan created email to dev team", zap.Error(err))
+			err = nil
+		}
+	}
```

Comment:
I probably should have clarified what I meant in my previous review - my feedback around not blocking the response was less specifically around how we handle the error, but was more around not blocking the response of this resolver based on the email being sent.

I think what we could do is wrap this code in an anonymous goroutine so that it runs asynchronously, that way we can respond as soon as we create the entity in the DB, and the email will fire off / log errors on its own.

Reply:
I think this would be good too, and next step would be offloading this to faktory. I happened to test this and make the error pop up, and not block model plan creation which is great. I do think that offloading it further to make it asynchronous is important here.

Reply:
👍